index.fa.md

```diff
@@ -0,0 +1,2474 @@
+---
+
+{% include share-links.html %}
+
+_2021-03-23_
+
+ریچارد ام. استالمن ، که اغلب با نام RMS شناخته می شود ، برای مدت طولانی یک نیروی خطرناک در جامعه نرم افزار آزاد بوده است. او خود را زن ستیز ، توانمند و ترنسفوبیک نشان داده است ، از جمله دیگر اتهامات جدی به بی ادب بودن است. این نوع باورها جایی در نرم افزار رایگان ، حقوق دیجیتال و جوامع فناوری ندارند. با اعاده مجدد وی به عضو هیئت مدیره بنیاد نرم افزار آزاد ، ما خواهان کنار رفتن کل هیئت FSF و برکناری RMS از کلیه مناصب رهبری هستیم.
+
+ما ، امضا شدگان ، به ضرورت خودمختاری دیجیتال و نقش پرقدرت آزادی کاربر در حمایت از حقوق بنیادی انسانی خود ایمان داریم. برای تحقق وعده هر آنچه آزادی نرم افزار امکان پذیر است ، باید تغییرات اساسی در جامعه ایجاد شود. ما به حال و آینده ای اعتقاد داریم که فن آوری به مردم توانمندی  (و نه ظلم) می دهد. ما می دانیم که این تنها در جهانی امکان پذیر است که فناوری برای احترام به حقوق ما در اساسی ترین سطح خود ساخته شود. در حالی که این ایده ها توسط ریچارد ام. استالمن به نوعی رواج یافته است ، اما او به جای ما صحبت نمی کند. ما از اعمال و نظرات وی چشم پوشی نمی کنیم. ما رهبری وی یا رهبری بنیاد نرم افزار آزاد را که امروز برقرار است ، تأیید نمی کنیم. 
+
+تا کنون تحمل کافی عقاید و رفتارهای نفرت انگیز RMS وجود داشته است. ما نمی توانیم ادامه دهیم که یک نفر معنای کار ما را خراب کند. جوامع ما فضایی برای افرادی مانند ریچارد ام. استالمن ندارند و ما با رفتارهای وی ، دادن نقش رهبری به او و یا در غیر اینصورت پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ، ادامه نخواهیم داد. 
+
+**ما خواهان حذف کل اعضای بنیاد نرم افزار آزاد هستیم.** اینها افرادی هستند که سالها RMS را فعال و توانمند کرده اند. آنها با اجازه دادن به او برای عضویت مجدد در شورای FSF این امر را دوباره نشان می دهند. وقت آن است که RMS از نرم افزار آزاد ، اخلاق فناوری ، حقوق دیجیتال و جوامع فناوری عقب نشینی کند ، زیرا او نمی تواند رهبری مورد نیاز ما را تأمین کند. **ما همچنین خواهان برکناری ریچارد ام. استالمن از کلیه مناصب رهبری ، از جمله پروژه GNU هستیم.**
+
+ما از افرادی که در موقعیتی هستند اصرار داریم که این کار را انجام دهند تا حمایت از بنیاد نرم افزار آزاد را متوقف کنند. از مشارکت در پروژه های مربوط به FSF و RMS خودداری کنید. در رویدادهای FSF ، یا رویدادهایی که از RMS و مارک عدم تحمل وی استقبال می کنند ، صحبت نکنید و در آنها شرکت نکنید. ما از همکاران پروژه های نرم افزار رایگان می خواهیم تا در برابر تعصب و نفرت در پروژه های خود موضع بگیرند. در حالی که این کارها را انجام می دهید ، به این جوامع و FSF بگویید که چرا این کار را انجام می دهید. 
+
+[ما چندین واقعه عمومی راجع به رفتار RMS شرح داده ایم][1]. برخی از ما داستان های خود را در مورد RMS و تعاملات خود با او داریم ، مواردی که در موضوعات ایمیل یا ویدیو ثبت نشده اند. امیدواریم آنچه را که به اشتراک گذاشته شده است بخوانید و آسیبی را که وی به جامعه ما و دیگران وارد کرده است در نظر بگیرید. 
+
+[1]: https://rms-open-letter.github.io/appendix
+
+To sign, please email digitalautonomy at riseup.net or [submit a pull request](https://github.com/rms-open-letter/rms-open-letter.github.io/pulls).
+
+Signed,
```

Comment:
Please drop everything from here down, we aren't keeping copies of the signatures in the translations.

Reply:
Done.